README.md

```diff
@@ -111,9 +127,11 @@ sudo yum-config-manager --add-repo https://copr.fedoraproject.org/coprs/petersen
 sudo yum install ghc cabal-install
```

Comment:
Does not work even with the smallest of margins :frowning:

![margins](https://cloud.githubusercontent.com/assets/5645741/6156837/df63c934-b262-11e4-97ff-24370011144e.png)